### ISSUE TITLE
invalid Range header for sort

### ISSUE THREAD

**Issue by keturn** [martyzz1/heroku3.py#12]
When I try to do the "list releases" example in the README:

```.py
app.releases(order_by='version', limit=10, sort='desc')
```

the API responds with an error:
```.json
{
  "message": "Invalid `Range` header. Please use format like `id ..; max=10, order=desc`. ", 
  "id": "bad_request"
}
```

The Range header in the request is:
```
version ..; max=10; order=desc
```

semicolon instead of a comma?

**Comment by martyzz1** [comment]
Could be. I implemented the range feature whilst it was in beta from
heroku. I suspect they've changed it. If you want to submit a patch I can
merge it, but I'm away from my computer for a couple of days...

**Comment by martyzz1** [comment]
This is now fixed in 5c46113
